### PR TITLE
Fixes #14182: Remove initial promises from rudder agent package

### PR DESCRIPTION
**Pull request by peckpeck**
https://issues.rudder.io/issues/14182

### CODE REVIEW

File: rudder-webapp/SOURCES/Makefile [hunk 1]
```diff
@@ -99,6 +99,15 @@ jetty:
...
 	test `find . -name \*.rej | wc -l` = 0
 
+initial-promises: ./rudder-sources
+	rm -rf ./initial-promises/
+	cp -r ./rudder-sources/rudder-techniques/initial-promises/node-server/ ./initial-promises
+
+initial-ncf: ./rudder-sources ./initial-promises
+	rm -rf ./initial-ncf
+	cp -r ./rudder-sources/ncf/tree ./initial-ncf
+	# retreive the rudder specific ncf.conf
```
Comment (amousset) [inline rudder-webapp/SOURCES/Makefile]: you can remove this now

**Comment by peckpeck** [comment]
Commit modified

**Comment by amousset** [comment]
@peck needs rebase

**Comment by peckpeck** [comment]
PR rebased

File: rudder-agent/SOURCES/rudder-agent-postinst [hunk 2]
```diff
@@ -200,19 +200,7 @@ then
   RESTORE_SERVER_ROLES_BACKUP=1
 fi
 
-# Copy initial promises if there aren't any already or,
```
Comment (amousset) [inline rudder-agent/SOURCES/rudder-agent-postinst]: Don't we need another fallback? (like remove policies if they are corrupted?)

Comment (peckpeck) [reply rudder-agent/SOURCES/rudder-agent-postinst]: a reset ?

**Comment by fanf** [comment]
ping @peckpeck (a chat with @amousset may be needed)

**Comment by ncharles** [comment]
you should remove the copy l322 and 323 (can't link it in conversation somehow) https://github.com/Normation/rudder-packages/blob/715f68914c6dab2f93ee229998ea59f58bfffb28/rudder-agent/SOURCES/Makefile#L322

**Comment by peckpeck** [comment]
Commit modified

**Pull request merged**